CppExport/src/dependency_analysis/DependencyAnalyzer.cpp

```diff
@@ -89,4 +89,34 @@ QList<DependencyComposite*> DependencyAnalyzer::mergeUnits(QList<DependencyUnit*
 	return nameToCompositeMap.values();
 }
 
+QHash<DependencyComposite*, QSet<DependencyComposite*>>
+DependencyAnalyzer::dependencies(const QList<DependencyComposite*>& composites)
```

Comment:
Shouldn't this method be in `DependecyComposite`? Doesn't it compute the dependencies of all composites? If we move it to composite, the signature will also become a little bit saner.